pkg/controller/core/clusterqueue_controller.go

```diff
@@ -571,7 +578,7 @@ func (r *ClusterQueueReconciler) shouldUpdatePendingWorkloadStatus(cq *kueue.Clu
 
 func (r *ClusterQueueReconciler) Start(ctx context.Context) error {
 	// Taking snapshot of cluster queue is enabled when maxcount non-zero
```

Comment:
Move the comment to the body of the function, or above the function definition

Reply:
done